### DISCUSSION TITLE
Add ROCm wheels

### DISCUSSION THREAD

**Discussion by kunaltyagi** [pyg-team/pytorch_geometric#5372] [General]
I noticed that https://data.pyg.org/whl/ doesn't have wheels for ROCm. Any plans to do so? I saw some prev discussions which mentioned no immediate plans

Is it possible for someone else to contribute the wheels? If so, how?

**Comment by rusty1s** [comment]
We build all our wheels with github actions. In case there exist a github runner that allows us to build ROCm wheels, this should be trivial to add. It likely only requires changes to our build script, see https://github.com/rusty1s/pytorch_scatter/blob/master/.github/workflows/building.yml.

**Reply by kunaltyagi** [reply]
Thanks. I tried to change the torch wheel url, and hit upon some issues, so I've raised a question over at pytorch discourse. I'll follow-up on this after [this question](https://discuss.pytorch.org/t/missing-filter-for-rocm-for-downloading-wheels/161050) is answered.

**Reply by rusty1s** [reply]
Thank you! Keep me posted.
